docs/resources/logs_custom_pipeline.md

```diff
@@ -185,6 +186,7 @@ The following arguments are supported:
   - `source_type`: (Required) Defines where the sources are from (log `attribute` or `tag`).
   - `target`: (Required) Final `attribute` or `tag` name to remap the sources.
   - `target_type`: (Required) Defines if the target is a log `attribute` or `tag`.
+  - `target_format`: (Optional) If the target of the remapper is an attribute, try to cast the value to a new specific type. If the cast is not possible, the original type is kept. `string`, `integer`, or `double` are the possible types.
```

Comment:
Maybe you can make it more explicit that users should only set the target_format if the target_type is an attribute. It's already said, but for this kind of valid / invalid values it's better to be super explicit and not being afraid of being repetitive.

Reply:
thank you for pointing this out